projects/ngneat/error-tailor/src/lib/control-error.component.ts

```diff
@@ -1,6 +1,12 @@
 import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, TemplateRef } from '@angular/core';
 import { ValidationErrors } from '@angular/forms';
 
+export interface IControlErrorComponent {
```

Comment:
Please remove the `I` prefix. You can change `ControlErrorComponent` to `DefaultControlErrorComponent`